src/Symfony/Component/Security/Http/Tests/Firewall/AccessListenerTest.php

```diff
@@ -12,198 +12,331 @@
 namespace Symfony\Component\Security\Http\Tests\Firewall;
 
 use PHPUnit\Framework\TestCase;
+use Symfony\Component\Security\Core\Authentication\Token\AbstractToken;
+use Symfony\Component\Security\Core\Authorization\AuthorizationChecker;
 use Symfony\Component\Security\Http\Firewall\AccessListener;
 
 class AccessListenerTest extends TestCase
 {
+    private $request;
+    private $accessMap;
+    private $token;
+    private $tokenStorage;
+    private $accessDecisionManager;
+    private $event;
+    private $authenticationManager;
+    private $authorizationChecker;
+
+    public function setUp()
```

Comment:
I'm not sure about this change. We prefer not doing refactos of the test suite. Could you revert these changes?

Reply:
No problem, I'll revert refactoring.